server/src/main/java/org/opensearch/index/SlowLogLevel.java

```diff
@@ -54,7 +54,7 @@ public static SlowLogLevel parse(String level) {
         return valueOf(level.toUpperCase(Locale.ROOT));
     }
 
-    boolean isLevelEnabledFor(SlowLogLevel levelToBeUsed) {
+    public boolean isLevelEnabledFor(SlowLogLevel levelToBeUsed) {
```

Comment:
We should move this enum out of the `index` package, since it's no longer just an index-level setting. Maybe move it to `org.opensearch.common.logging`?

Reply:
Yes, that makes sense.